cmd/completion.go

```diff
@@ -0,0 +1,233 @@
+package cmd
```

Comment:
Add the project copyright notice please.

Reply:
no problem